source/lib/index.ts

```diff
@@ -28,6 +35,7 @@ const findTestFiles = async (typingsFile: string, options: Options & {config: Co
 	const tsxTestFile = typingsFile.replace(/\.d\.ts$/, '.test-d.tsx');
 	const testDir = options.config.directory;
 
+	//@ts-ignore
```

Comment:
```suggestion
	// @ts-expect-error
```

The difference is that `// @ts-ignore` comments will do nothing if the following line is error-free. We want to avoid `// ts-ignore` as it may suppress errors you may originally not intend too (which is dangerous sometimes).

Similar for further `// @ts-ignore`s

Reply:
^TIL. I'll change 'em.